www/layouts/resources/single.html

```diff
@@ -0,0 +1,11 @@
+{{ define "main" }}
+{{ block "header" . }}
+  {{ partialCached "header" . }}
+{{ end }}
+<div class="container mt-5">
+  <div class="rounded mb-2">
+    {{ partial "content_header.html" . }}
+  </div>
+  {{ partial "resource_layout.html" . }}
```

Comment:
Take it or leave it, but I think a better name for this partial might be `resource_body.html`.